From 564364e2e414dec34aade7f2c723f4a0a3c875d8 Mon Sep 17 00:00:00 2001
From: sai surya teja <sssteja@habana.ai>
Date: Mon, 23 Jan 2023 17:24:01 +0200
Subject: [PATCH] [SW-0] gaudi2: security: fix address in comments of secure_hbw_common

This patch fixes the address value for SRAM_RSVD_BASE in comments of
secure_hbw_common function.

Change-Id: Iaf57e54fd8dfb2eb097e5b50efac358aa674d01e
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index de2383d..279798a 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -993,7 +993,7 @@
 
 	/*
 	 * Block Reserved space to HBM
-	 * 0x1000FFFFFD000000 - 0x1000FFFFFFFFF000
+	 * 0x1000008000000000 - 0x1000FFFFFFFFF000
 	 */
 	min = SRAM_RSVD_BASE;
 	max = SRAM_END_EXC_TOP4K;
